02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
```

Comment:
This function currently doesn't contain any code. Since it is the one which should be generating our password (when it is called on line 49), we will not get a positive result.